test/helpers/api-integration/v3/object-generators.js

```diff
@@ -121,15 +123,16 @@ async function _upgradeToGroupPlan (groupLeader, group) {
 // invitees: an array of user objects that correspond to the invitees of the group
 // leader: the leader user object
 // group: the group object
-export async function createAndPopulateGroup (settings = {}) {
+export async function createAndPopulateGroup (settings = {}, update = {}) {
   const numberOfMembers = settings.members || 0;
   const numberOfInvites = settings.invites || 0;
   const upgradeToGroupPlan = settings.upgradeToGroupPlan || false;
   const { groupDetails } = settings;
+  const { groupDetailsUpdate } = update;
```

Comment:
is this needed? why can't `groupDetails ` be used?

Reply:
Admittedly, the approach I was taking when updating the test was sloppy due to being quite unfamiliar with the code base. After I got more familiar with it, I noticed the approach I was taking was pretty silly. This has all been amended